File: .ProgramingBasics/ComplexConditionalStatements/04.FruitOrVegetable/src/FruitOrVegetable.java
import java.util.Scanner;

public class FruitOrVegetable {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String input = scanner.nextLine();
        boolean isFruit = input.equals("banana") || input.equals("apple") || input.equals("kiwi") ||
                input.equals("cherry") || input.equals("lemon") || input.equals("grapes");

        boolean isVegetable = input.equals("tomato") || input.equals("cucumber") ||
                input.equals("pepper") || input.equals("carrot");

        if (isFruit) {
            System.out.println("fruit");
        }
        else if (isVegetable) {
            System.out.println("vegetable");
        }
        else {
            System.out.println("unknown");
        }
    }
}

File: ProgramingBasics/ComplexConditionalStatements/04.FruitOrVegetable/04.FruitOrVegetable/FruitOrVegetable.cs
using System;

namespace _04.FruitOrVegetable
{
    public class FruitOrVegetable
    {
        public static void Main(string[] args)
        {
            var input = Console.ReadLine();
            var isFruit = input == "banana" || input == "apple" || input == "kiwi" || input == "cherry" || input == "lemon" || input == "grapes";
            var isVegerable = input == "tomato" || input == "cucumber" || input == "pepper" || input == "carrot";

            if (isFruit)
            {
                Console.WriteLine("fruit");
            }
            else if (isVegerable)
            {
                Console.WriteLine("vegetable");
            }
            else
            {
                Console.WriteLine("unknown");
            }
        }
    }
}
